languages/javalang/builder-renderer/commercetools-java-client/src/main/java/client/ApiHttpMethod.java

```diff
@@ -1,5 +1,5 @@
 package client;
 
-public enum HttpMethod {
+public enum ApiHttpMethod {
```

Comment:
Yeah, me too, but the thing is that java 11 client (and other http client libraries) already contain these names (HttpHeaders, HttpClient, Http... ), so I changed ours so that there is no confusion. I can return them back to the old ones if its fine to have the same class names.